src/modules/AutoinstPartPlan.ycp

```diff
@@ -375,6 +389,15 @@
                 if( pe["type"]:`x == `extended )
                     continue;
                 map new_pe = $[];
+                // Handling /dev/nfs containers (bnc#986124)
+                if( pe["type"]:`x == `nfs ) {
+                  new_pe["type"] = pe["type"]:`nfs;
+                  new_pe["device"] = pe["device"]:"";
+                  new_pe["mount"] = pe["mount"]:"";
+                  new_pe["fstopt"] = pe["fstopt"]:"";
+                  partitions=add(partitions,new_pe);
```

Comment:
Please keep spaces around `=`, and keep the 4 spaces indentation like the rest of the code.